pyproject.toml

```diff
@@ -64,17 +91,22 @@ include = ["dagfactory"]
 [tool.hatch.build.targets.wheel]
 packages = ["dagfactory"]
 
-[[tool.hatch.envs.tests.matrix]]
-python = ["3.9", "3.10", "3.11", "3.12"]
-airflow = ["2.8", "2.9", "2.10"]
+[tool.pytest.ini_options]
+filterwarnings = ["ignore::DeprecationWarning"]
+minversion = "6.0"
 
-[tool.hatch.envs.tests.scripts]
-static-check = " pre-commit run --files dagfactory/*"
+######################################
+# THIRD PARTY TOOLS
+######################################
 
 [tool.black]
 line-length = 120
 target-version = ['py39', 'py310', 'py311', 'py312']
 
+[tool.isort]
```

Comment:
isort pre-commit hook is not in our pre-commit config file at the moment if we are adding this configuration, I think we should add the hook also in https://github.com/astronomer/dag-factory/blob/main/.pre-commit-config.yaml

Reply:
made it as part of tool.ruff.lint instead in the pyproject.toml so that it gets checked via the ruff pre-commit hook